definitions/res-gitrepository/definition.yml

```diff
@@ -0,0 +1,5 @@
+domain: RES
+type: GITREPOSITORY
```

Comment:
nit: Should we GIT_REPOSITORY instead?

Reply:
I don't have a strong opinion, these are not meant to be user visible anyway

Reply:
I only went with this approach since it looked like a lot of the AWS related types did the same (i.e. `AWSIAMOPENIDPROVIDER`).Totally happy to change this if we like the `GIT_REPOSITORY` instead.

Reply:
whatever you prefer :D the names where without the `_` because the limit was too low before. I'm ok with any variation.